src/js/visual/image.js

```diff
@@ -195,7 +195,8 @@ ripe.Image.prototype.cancel = async function(options = {}) {
  * it should stop responding to updates so that any necessary
  * cleanup operations can be executed.
  */
-ripe.Image.prototype.deinit = function() {
+ripe.Image.prototype.deinit = async function() {

```

Comment:
@NFSS10 this is not enough, it has to be replicated to all other `deinit` and the functions that call them.

Reply:
@NFSS10 still unresolved.